devices.js

```diff
@@ -9249,6 +9249,15 @@ const devices = [
         description: 'Zigbee A60 filament bulb 6.3W',
         extend: generic.light_onoff_brightness,
     },
+    {
+        zigbeeModel: ['ZB-MotionSensor-D0003'],
+        model: 'ZS1100400-IN-V1A02',
+        vendor: 'Linkind',
+        description: 'PIR Motion Sensor, Wireless Motion Detector',
+        supports: 'occupancy',
+        fromZigbee: [fz.linkind_occupancy],
```

Comment:
Please use https://github.com/Koenkk/zigbee-herdsman-converters/blob/915709676fd27220920bc504d668339d6cb10bc4/converters/fromZigbee.js#L682 (fz.ias_occupancy_alarm_1)

Reply:
:+1:  Thanks, I didn't realize that was what I wanted. Done.